qark/plugins/Manifest_File_Checks.py

```diff
@@ -1,22 +1,71 @@
-import sys, os, re
-import qarkMain
-
-sys.path.insert(0, os.path.dirname(os.path.realpath(__file__)) + '../lib')
-
 from yapsy.IPlugin import IPlugin
 from plugins import PluginUtil
 from modules import common
 from lib.pubsub import pub
+import sys
+import os
+import re
+import qarkMain
+
+sys.path.insert(0, os.path.dirname(os.path.realpath(__file__)) + '../lib')
```

Comment:
We might still need this call above the other import to ensure that the `lib` directory is in the `PYTHONPATH` before trying to import other things. Import `sys` first, then use the `sys.path.insert...` line after it, and then keep on importing other modules.